### PR TITLE
Add steepcheck for lib/lrama/grammar/parameterizing_rules/**/*.rb

### PR DESCRIPTION
**Pull request by ydah**
This PR add steepcheck for lib/lrama/grammar/parameterizing_rules/**/*.rb

### CODE REVIEW

File: sig/lrama/grammar/parameterizing_rules/builder/list.rbs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module Lrama
+  class Grammar
+    class ParameterizingRules
+      class Builder
+        class List < Base
+          def build: -> Array[Rule]
```
Comment (yui-knk) [inline sig/lrama/grammar/parameterizing_rules/builder/list.rbs:6]: I think these sub classes inherit type definition of `#build` from `Base` class. Is this, and following type definition of `#build`, needed?

File: Steepfile [hunk 2]
```diff
@@ -8,6 +8,13 @@ target :lib do
   check "lib/lrama/grammar/code.rb"
   check "lib/lrama/grammar/counter.rb"
   check "lib/lrama/grammar/error_token.rb"
+  check "lib/lrama/grammar/parameterizing_rules/builder.rb"
```
Comment (Little-Rubyist) [inline Steepfile:11]: It would be better to write `"lib/lrama/grammar/parameterizing_rules"` to detect when files are added to the directory.

Comment (yui-knk) [reply Steepfile:11]: Sound reasonable.

Comment (ydah) [reply Steepfile]: Indeed it is. Thank you! I updated it.

File: sig/lrama/grammar/parameterizing_rules/builder/list.rbs (hunk 1 above)
Comment (ydah) [reply sig/lrama/grammar/parameterizing_rules/builder/list.rbs]: Indeed it is. Thank you! I deleted it.

**Comment by yui-knk** [comment]
Thank you!

**Pull request merged**